python_modules/dagster/dagster/_core/storage/db_io_manager.py

```diff
@@ -76,6 +77,18 @@ def delete_table_slice(
     @abstractmethod
     def get_select_statement(table_slice: TableSlice) -> str: ...
 
+    @staticmethod
+    @abstractmethod
```

Comment:
This shouldn't be abstract, right?

Reply:
oh yeah good catch, initially was going to have each connection implement this but realized it was unnecessary